app/src/main/java/com/spencerpages/collections/EarlyQuarters.java

```diff
@@ -0,0 +1,255 @@
+
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_5, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_5_STRING_ID, R.string.include_standing_quarters);
+
+        // Use the MINT_MARK_1 checkbox for whether to include 'P' coins
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_1, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_1_STRING_ID, R.string.include_p);
+
+        // Use the MINT_MARK_2 checkbox for whether to include 'D' coins
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_2, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_2_STRING_ID, R.string.include_d);
+
+        // Use the MINT_MARK_3 checkbox for whether to include 'S' coins
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_3, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_3_STRING_ID, R.string.include_s);
+
+        // Use the MINT_MARK_4 checkbox for whether to include 'O' coins
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_4, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_4_STRING_ID, R.string.include_o);
+
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_5, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_SHOW_MINT_MARK_5_STRING_ID, R.string.include_cc);
+    }
+
+    // TODO Perform validation and throw exception
+    @Override
+    public void populateCollectionLists(HashMap<String, Object> parameters, ArrayList<CoinSlot> coinList) {
+
+        Integer startYear = (Integer) parameters.get(CoinPageCreator.OPT_START_YEAR);
+        Integer stopYear = (Integer) parameters.get(CoinPageCreator.OPT_STOP_YEAR);
+        Boolean showOld = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_1);
+        Boolean showBust = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_2);
+        Boolean showSeated = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_3);
+        Boolean showBarber = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_4);
+        Boolean showStanding = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_5);
+        Boolean showP = (Boolean) parameters.get(CoinPageCreator.OPT_SHOW_MINT_MARK_1);
+        Boolean showD = (Boolean) parameters.get(CoinPageCreator.OPT_SHOW_MINT_MARK_2);
+        Boolean showS = (Boolean) parameters.get(CoinPageCreator.OPT_SHOW_MINT_MARK_3);
+        Boolean showO = (Boolean) parameters.get(CoinPageCreator.OPT_SHOW_MINT_MARK_4);
+        Boolean showCC = (Boolean) parameters.get(CoinPageCreator.OPT_SHOW_MINT_MARK_5);
+        int coinIndex = 0;
+
+        if (showOld && !showBust) {
+            coinList.add(new CoinSlot("Draped Bust", "", coinIndex++));
+            coinList.add(new CoinSlot("Capped Bust", "", coinIndex++));
+        }
+        if (showOld && !showSeated) {coinList.add(new CoinSlot("Liberty Seated", "", coinIndex++));}
+        if (showOld && !showBarber) {coinList.add(new CoinSlot("Barber", "", coinIndex++));}
```

Comment:
@aDesertRat I don't think these should have `!` - I'm guessing this was a typo, but wanted to confirm

```suggestion
        if (showOld && showBust) {
            coinList.add(new CoinSlot("Draped Bust", "", coinIndex++));
            coinList.add(new CoinSlot("Capped Bust", "", coinIndex++));
        }
        if (showOld && showSeated) {coinList.add(new CoinSlot("Liberty Seated", "", coinIndex++));}
        if (showOld && showBarber) {coinList.add(new CoinSlot("Barber", "", coinIndex++));}
```